src/c++/perf_analyzer/client_backend/mock_client_backend.h

```diff
@@ -92,6 +92,16 @@ class MockClientStats {
     }
 
     void HandleSeqRequest(uint64_t seq_id) { live_seq_ids_to_length[seq_id]++; }
+    void Reset()
+    {
+      // Note that live_seq_ids_to_length is explicitly not reset here.
+      // This is because we always want to maintain the true status of
+      // live sequences
```

Comment:
Why do we want to maintain the true status of live sequences? Also why does it look like there are two definitions of Reset()?

Reply:
Because that is what Triton will observe. If we break the rules about sequence start/end, then the server will assert. Likewise, if there IS a live sequence and we tell the mock client to forget about it, then when the manager legally sends the next request in that sequence the mock client would complain that it isn't a legal request when it really is.

This file has MockClientStats.Reset() and SeqStatus.Reset(). One calls the other.